dist/justified-layout.js

```diff
@@ -682,55 +653,70 @@ function computeLayout(itemLayoutData) {
 
 	return {
 		containerHeight: layoutData._containerHeight,
+		widowCount: layoutConfig._widowCount,
 		boxes: layoutData._layoutItems
 	};
 }
 
 /**
-* Create a new, empty row.
+* Takes in a bunch of box data and config. Returns
+* geometry to lay them out in a justified view.
 *
-* @method createNewRow
-* @return A new, empty row of the type specified by this layout.
-*/
-function createNewRow() {
+* @method covertSizesToAspectRatios
+* @param sizes {Array} Array of objects with widths and heights
+* @return {Array} A list of aspect ratios
+**/
+module.exports = function (input) {
+	var config = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];
```

Comment:
Great question. This was just an eslint update but I don't see why that needs to be so complex.